app/styles/components/input-field.css

```diff
@@ -0,0 +1,40 @@
+.input-field {
+  width: 100%;
+  margin-bottom: 1em;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.input-field label {
+  font-size: 1rem;
+  font-weight: bold;
+  padding: 10px 0;
+}
+
+.input-field input {
+  padding: 1em;
+  border: #666 1px solid;
+  border-radius: 5px;
+  width: 100%;
+}
+
+.input-field input:focus {
+  border: 1px solid rgb(0, 81, 255);
+  outline: 1px solid transparent;
+}
+
+.input-field > .error {
+  font-size: 0.8rem;
+  color: #e74c3c;
+}
+
+.help-msg {
+  font-size: 10px;
+  text-align: left;
+}
+
+.error-border {
+  border: 1px solid red !important;
```

Comment:
Let's avoid "important" for this simplistic case.

https://uxengineer.com/css-specificity-avoid-important-css/

Reply:
This is why it is marked as important 
![image](https://user-images.githubusercontent.com/56217868/144297261-5b413c7f-4d97-466f-aa9d-74174f668309.png)

Reply:
I would want to take a step back here and wonder "What does the error border help us do, and what does the focus ring help us do?"

I think the error border is just a quick way of noticing what is "wrong" on the page, especially when you are doing the final action e.g clicking the submit button.

The focus ring is there to tell you where you are typing. If there are 10 boxes, all having the red error border, how would a user distinguish where they are currently typing?

Reply:
Hmm... Makes sense, made the change